phpstan.neon

```diff
@@ -0,0 +1,69 @@
+parameters:
+    ignoreErrors:
+        #########
+        # Level 0
+        #########
```

Comment:
Tried to document each error as best I could.  Many of these could probably be fixed if we want to alter Doctrine's code (some would require changing API/signatures).